_po/ja/how-to/laravel.po

```diff
@@ -554,6 +830,14 @@ msgid ""
 "                                      </dt>\n"
 "                                    </div>"
 msgstr ""
+"                                  <dl class=\"sm:mt-8 grid grid-cols-1 gap-x-6 "
+"gap-y-2 grid-cols-1 sm:grid-cols-7 sm:gap-y-2 lg:gap-x-8\">\n"
+"                                    <div class=\"border-t border-gray-200\">\n"
+"                                      <dt class=\"text-sm sm:text-base sm:font-"
+"medium text-gray-900\">\n"
+"                                              ID\n"
+"                                      </dt>\n"
+"                                    </div>"
```

Comment:
Probably, this is needless.
Because this test is same as before translation text.